test/parallel/test-http-exceptions.js

```diff
@@ -30,16 +35,18 @@ const server = http.createServer(function(req, res) {
   res.end();
 });
 
+function onUncaughtException(err) {
+  console.log(`Caught an exception: ${err}`);
+  if (err.name === 'AssertionError') throw err;
+  countdown.dec();
+}
+
+process.on('uncaughtException',
+           common.mustCall(onUncaughtException, NUMBER_OF_EXCEPTIONS)
+);
```

Comment:
Can this be brought up to the previous line? The dangling `);` is awkward.

Reply:
sadly it cant be moved up since it will break the linter due to reaching the max limit of 80 characters in one line

Reply:
It's only 70 chars as far as I can tell?